0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -2,7 +2,7 @@
 coding: utf-8
 
 title: Crypto-Conditions
-docname: draft-thomas-crypto-conditions-00
+docname: draft-thomas-crypto-conditions-02
```

Comment:
The version should not have been 00 to begin with. The version at https://interledger.org/five-bells-condition/spec.html is numbered 01 already.